Use get_real_time instead of deprecated TimeVal

kinichiro opened the pull request:
Suppress these warnings.
nicola.vala:36.22-36.28: warning: `GLib.TimeVal' has been deprecated since 2.62
nicola.vala:36.17-36.18: warning: `GLib.TimeVal' has been deprecated since 2.62

kinichiro committed:
Use get_real_time instead of deprecated TimeVal

Suppress these warnings.
nicola.vala:36.22-36.28: warning: `GLib.TimeVal' has been deprecated since 2.62
nicola.vala:36.17-36.18: warning: `GLib.TimeVal' has been deprecated since 2.62

ueno reviewed (approved):
For this use-case, `get_monotonic_time` might be more appropriate.

kinichiro committed:
Use get_monotonic_time instead of deprecated TimeVal

Suppress these warnings.
nicola.vala:36.22-36.28: warning: `GLib.TimeVal' has been deprecated since 2.62
nicola.vala:36.17-36.18: warning: `GLib.TimeVal' has been deprecated since 2.62

kinichiro commented:
Agree. I had replaced get_real_time with get_monotonic_time.

ueno commented:
Thank you.
